kover-gradle-plugin/examples/android/dynamic/dyn/build.gradle.kts

```
@@ -0,0 +1,82 @@
+plugins {
+    id ("com.android.library")
+    id ("org.jetbrains.kotlin.android")
+    id ("org.jetbrains.kotlinx.kover")
+}
+
+android {
+    namespace = "kotlinx.kover.test.android"
+
+    compileSdk = 32
+
+    defaultConfig {
+        minSdk = 21
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = true
+        }
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+}
+
+dependencies {
+    implementation("androidx.core:core-ktx:1.8.0")
+    implementation("androidx.appcompat:appcompat:1.5.0")
+    testImplementation("junit:junit:4.13.2")
+}
```

You must add the base (:app) module dependency here. 

```suggestion
    implementation(project(":app"))
}
```